hw2/02-search.js

```diff
@@ -1,3 +1,85 @@
 // Add your code here
 
-userInput.addEventListener('click', handleClick);
+document.getElementById("searchButton").addEventListener("click", handleClick);
```

Comment:
I see you also noticed the error in the original event listener. Caused some errors before I figured that one out